### DISCUSSION TITLE
Add fields in error in error email on Nebula Logger exception.

### DISCUSSION THREAD

**Discussion by BastienRaoul** [jongpie/NebulaLogger#588] [Ideas]
Hello @jongpie,

When an error occurs in Nebula Logger, an email is sent with the error message to the apex email recipients.
Currently, in this email we only have access to the error message which may not be sufficient to debug the issue.
This could be improved by adding the fields that are in error with the `getFields` method.

**Comment by alanjaouen** [comment]
see https://github.com/jongpie/NebulaLogger/pull/591

**Reply by jongpie** [reply]
@BastienRaoul this enhancement is [now available in `v4.12.1`](https://github.com/jongpie/NebulaLogger/releases/tag/v4.12.1) . Many thanks to @alanjaouen for working on this in #591 👏
